SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/Storage.cs

```diff
@@ -238,8 +222,8 @@ public Task<string> GetLogsAsync(string channelName, int limit, List<Log> logs)
                     {
                         AppCenterLog.Error(AppCenterLog.LogTag, "Cannot deserialize a log in storage", e);
                         failedToDeserializeALog = true;
-                        _storageAdapter.DeleteAsync<LogEntry>(row => row.Id == entry.Id)
-                            .GetAwaiter().GetResult();
+                        var values = new List<object> { entry.Id };
```

Comment:
Is `var values` used anywhere? I can't find it.

Reply:
That's a good point :) Was left undeleted after refactor